packages/web-react/src/components/FileUploader/demo/index.tsx

```diff
@@ -0,0 +1,52 @@
+// Because there is no `dist` directory during the CI run
+/* eslint-disable import/no-extraneous-dependencies, import/extensions, import/no-unresolved */
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment, import/extensions, import/no-unresolved
+// @ts-ignore: No declaration file
+import icons from '@lmc-eu/spirit-icons/dist/icons';
```

Comment:
I think that `import icons from '@lmc-eu/spirit-icons'` should work as well 🤔 In this case Lerna should take care of binding those packages together. But please, try it first. And also try to locally build the app.

Reply:
it is not working for me with just `@lmc-eu/spirit-icons` import

Reply:
Ok, leave it as it is. Thanks :-)